### ISSUE TITLE
Discrepancy in results buoy in wave 

### ISSUE THREAD

**Issue by ewoudVL13** [FloatingArrayDesign/MoorDyn#272]
Hello everybody,
I am currently working on an model comparison between MoorDyn V2 and OrcaFlex. 
I ran a model of a line with 2 fixed end points in a regular wave and the results there are almost the same.  
![image](https://github.com/user-attachments/assets/11b80f60-b198-4942-a005-7b63afcc9a1b)
Now I use the same wave input or a buoy with a center of mass that is lowered. (input files below)(wave amplitude 4m, period 8.33s)
[buoy_mass.txt](https://github.com/user-attachments/files/17750485/buoy_mass.txt)
[water_grid.txt](https://github.com/user-attachments/files/17750486/water_grid.txt)
[wave_frequencies.txt](https://github.com/user-attachments/files/17750487/wave_frequencies.txt)
But here something strange happens. The buoy is moving backwards. The usefful results are shown below. 
![image](https://github.com/user-attachments/assets/ee717b44-5ff6-4f71-9bdb-3dae113f6ba1)
![image](https://github.com/user-attachments/assets/a3b6de94-0eba-42ee-a945-bd18e9546f3c)
![image](https://github.com/user-attachments/assets/41947fcb-3ae5-491e-aae6-95b687f5e77d)
The buoy does not lay flat and for a basic calculation no cd or ca coefficients are included.
When only a current acts on the buoy the models show the same results.
Any ideas?
The buoy has a center of mass 2.67m above the bottom of the buoy. I think this is correctly implemented no?

**Comment by RyanDavies19** [comment]
Hi @ewoudVL13, 

Sorry for the slow response here. It seems like your buoy has no hydrodynamic properties (Rod: Cd, Ca, CdEnd, and CaEnd). Is this intentional? My guess is this is causing the discrepancies.

**Comment by ewoudVL13** [comment]
@RyanDavies19 Yeah it was done on purpose to see how the to programs act when only buoyancy is in the play.

**Comment by ewoudVL13** [comment]
Yeah indeed I found that during the simulations. Dou you know something about the influence of wave height or wave period on the results?

**Comment by RyanDavies19** [comment]
I'm not sure I can give specifics, but without the hydrodynamic forcing I would think the buoy would drift towards the anchor point rather than stay where it is. It could also be good to check decay tests of the buoy with no mooring, to make sure the dynamics are matching between the two (if results differ when including hydro stuff, then setting the water density to zero will determine if the kinematics are right). With no mooring and a cylindrical shape, pitch and heave should be sufficient.

**Comment by RyanDavies19** [comment]
Closing due to inactivity
